layouts/partials/blocks/testimonials.html

```diff
@@ -0,0 +1,59 @@
+{{ with .block }}
+  <section
+    class="min-h-[65vh] bg-black bg-opacity-10 py-20"
+    data-ga-category="links"
+  >
+    <div class="mx-auto max-w-6xl px-5">
+      <div class="lg:flex">
+        <h2
+          class="lg:sideways-heading mb-4 text-4xl font-black uppercase leading-none text-s-7 opacity-40 md:text-6xl lg:mb-0 lg:rotate-180"
+        >
+          {{ .title }}
+        </h2>
+        <div class="mt-8 flex" x-data="carousel">
+          {{ with .entries }}
+
+            <div class="hidden h-64 w-20 md:flex">
+              <span
+                class="h-12 w-12 cursor-pointer self-center"
+                @click="slide('left', 1)"
+              >
```

Comment:
The left button here shows up even at position zero. The Alpine Components one is greyed out at zero, so that's a good reason to switch.